### PR TITLE
fix(pci.storages.databases): datatr-299 - display correct value for vcores number

### PR DESCRIPTION
**Pull request by jperchoc**
DATATR-299

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #datatr-299
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Fix number of cores displayed in order funnel summary

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html [hunk 1]
```diff
@@ -386,8 +386,8 @@ <h2 data-translate="pci_database_add_step7_title"></h2>
                                     data-ng-pluralize
                                     data-count="$ctrl.model.flavor.core"
                                     data-when="{
-                                        '1': ('pci_database_add_spec_flavor_CPU_one' | translate: { cpu: $ctrl.model.flavor.core }),
-                                        'other': ('pci_database_add_spec_flavor_CPU_many' | translate: { cpu: $ctrl.model.flavor.core }),
+                                        '1': ('pci_database_add_spec_flavor_CPU_one' | translate: { cpu: {} }),
```
[the remark this answers is not in the document]
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html:389]: Translations are needed here, as the `cpu` var is used in the translation. The empty object `{}` passed in the `translate` filter is replaced by the `data-count` value by the `ng-pluralize` directive.
See documentation here: https://docs.angularjs.org/api/ng/directive/ngPluralize#examples

[the remark this answers is not in the document]
Comment (darsene) [reply packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html:389]: Yes I know.
My Point was to remove translation args if they are not used anymore

```suggestion
                                        '1': ('pci_database_add_spec_flavor_CPU_one' | translate),
```  
```
"pci_database_add_spec_flavor_CPU_one": "{} vCore",
 "pci_database_add_spec_flavor_CPU_many": "{} vCores",
```

[the remark this answers is not in the document]
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html:389]: I see. It works indeed, but then we'll have to request translations once again. I also fail to see how it is an improvement, as the code I wrote is basically the same as here:

- https://github.com/ovh/manager/blob/master/packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/billing/groupRepayments/group-repayments.html#L65
- https://github.com/ovh/manager/blob/master/packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/billing/calledFees/called-fees.html#L38
- https://github.com/ovh/manager/blob/master/packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/billing/calledFees/called-fees.html#L38
- https://github.com/ovh/manager/blob/master/packages/manager/modules/sms/src/sms/options/blacklist/telecom-sms-options-blacklist.html#L52

I'll ask Thibaud (who submitted the issue) what we should do here.

[the remark this answers is not in the document]
Comment (thild42) [reply packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html:389]: Hey @darsene, @jperchoc,
How much I value code simplification and technical debt reduction, this time the drawbacks, namely having to do a re-translation, seems to exceed the benefits.
I would suggest to stick with the current code using empty parameter to not have to redo the translation.

[the remark this answers is not in the document]
Comment (anooparveti) [reply packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html:389]: Hello @thild42, @jperchoc,
I would like you to remove the empty params even if it means redoing the trans. The suggested change simplifies the code and make it more readable.

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Comment by anooparveti** [comment]
Hello @jperchoc,
I see that the PR is closed. If the branch is not required anymore, may I request you to delete the branch on origin?
Thanks!

**Comment by anooparveti** [comment]
Deleting the branch since the PR was closed by the author.

**Review by abullet33 — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**